proposed/extended-coding-style-guide.md

```diff
@@ -111,7 +111,7 @@ tabs for indenting.
 
 PHP [keywords][] MUST be in lower case.
 
-The PHP types and keywords `array`, `int`, `true`, `object`, `float`, `false`, `mixed`,
+The PHP types and keywords `new`, `array`, `int`, `true`, `object`, `float`, `false`, `mixed`,
```

Comment:
..maybe we can improve this line, linking to https://secure.php.net/manual/en/reserved.other-reserved-words.php, so we can indirectly list them all, even future ones.